setup.py

```diff
@@ -40,18 +40,18 @@
         'invenio-theme>=1.0.0,<1.1.0',
     ],
     'auth': [
-        'invenio-access>=1.0.1,<1.1.0',
-        'invenio-accounts>=1.0.1,<1.1.0',
+        'invenio-access>=1.0.2,<1.1.0',
+        'invenio-accounts>=1.0.2,<1.1.0',
         'invenio-oauth2server>=1.0.1,<1.1.0',
         'invenio-oauthclient>=1.0.0,<1.1.0',
         'invenio-userprofiles>=1.0.1,<1.1.0',
     ],
     'metadata': [
-        'invenio-indexer>=1.0.0,<1.1.0',
+        'invenio-indexer>=1.0.1,<1.1.0',
         'invenio-jsonschemas>=1.0.0,<1.1.0',
         'invenio-oaiserver>=1.0.0,<1.1.0',
         'invenio-pidstore>=1.0.0,<1.1.0',
-        'invenio-records-rest>=1.1.0,<1.3.0',
+        'invenio-records-rest>=1.1.1,<1.2.0',
```

Comment:
Note, here I'm staying with ``<1.2.0`` and not ``<v1.3.0`` which was merged in master

Reply:
I think `1.1.1` was never released, travis was failing, what about `1.1.2` as minimum?